articlequality/__init__.py

```diff
@@ -7,6 +7,9 @@
 from .about import (__author__, __author_email__, __description__, __name__,
                     __url__, __version__)
 
-__all__ = [Extractor, TemplateExtractor, extract_from_text, extract_labelings,
-           extract_text, fetch_text, score, __name__, __version__, __author__,
-           __author_email__, __description__, __url__]
+__all__ = [
+        'Extractor', 'TemplateExtractor', 'extract_from_text',
+        'extract_labelings', 'extract_text', 'fetch_text', 'score',
+        __name__, __version__, __author__, __author_email__,
+        __description__, __url__
```

Comment:
I think these need to be strings too.

Reply:
good catch, thanks! -- these vars resolve to strings at runtime which meant the issue didn't show up on travis